.env.dist

```diff
@@ -0,0 +1,5 @@
+MODE=ONLINE
```

Comment:
> I think this could be helpful although I'm not sure it's necessary.

For sure can you help me understand why it's not necessary? We need 1 image that can be run in four different configurations of these values. How will you support that?

Reply:
Does my last comment below clear any of this up?

Reply:
>  All in all this is an optional enhancement should anyone want to use it.

I thought your goal was to start consolidating the different configs we have strewn about our package. 

I am **not aligned** with this _"optional enhancement"_ as you call it; adding more redundant configuration just adds to the problems discussed in #24.

I mean, look at all these current places we have some config values defined;

- [Makefile](https://github.com/rosetta-dogecoin/rosetta-dogecoin/blob/53a1733deba341965fb461705c2c2ba6b0ce6aec/Makefile#L35), 
- [configuration/configuration.go](https://github.com/rosetta-dogecoin/rosetta-dogecoin/blob/main/configuration/configuration.go#L40-L66), 
- [dogecoin/configuration.go](https://github.com/rosetta-dogecoin/rosetta-dogecoin/blob/main/dogecoin/configuration.go#L34-L47), 
- [dogecoin/params.go](https://github.com/rosetta-dogecoin/rosetta-dogecoin/blob/main/dogecoin/params.go#L45-L51), 
- [dogecoin/types.go](https://github.com/rosetta-dogecoin/rosetta-dogecoin/blob/main/dogecoin/types.go#L24-L44), 


I'm of the opinion that all of this configuration should be consolidated & structured in a known location - this PR goes against that and just adds complexity.

I'd ask others to review if you're looking for approval or switch to an approach that aligns with consolidating our configs as per #24.

Reply:
Ok.